src/html/template-parts/sections/newsletter/newsletter.html

```diff
@@ -0,0 +1,22 @@
+<section class="newsletter">
+    <div class="container">
+        <div class="newsletter-inner">
+            <div class="newsletter-info">
+                <div class="newsletter-title">
+                    Sign up for newsletter
+                </div>
+                <div class="newsletter-text">
+                    Cu qui soleat partiendo urbanitas. Eum aperiri indoctum eu, homero alterum.
+                </div>
+            </div>
+            <div class="newsletter-email">
+                <fieldset class="email-inner">
```

Comment:
You have fieldset tag, but have no main here - form tag. Fieldset is a part of the form. And every form must be html form, not div.